test/system/src/test/java/io/pravega/test/system/ReaderCheckpointTest.java

```diff
@@ -230,12 +230,15 @@ private void verifyEvents(final List<EventRead<Integer>> events, int startInclus
         Supplier<Stream<Integer>> streamSupplier = () -> events.stream().map(i -> i.getEvent()).sorted();
         IntSummaryStatistics stats = streamSupplier.get().collect(Collectors.summarizingInt(value -> value));
 
-        assertTrue("Check for first event", stats.getMin() == startInclusive);
-        assertTrue("Check for last event", stats.getMax() == endExclusive - 1);
+        assertTrue(String.format("Check for first event: %d, %d", stats.getMin(), startInclusive),
+                stats.getMin() == startInclusive);
+        assertTrue(String.format("Check for last event: %d, %d", stats.getMax(), endExclusive),
```

Comment:
Will it be good to print the second string endExclusive-1, instead of endExclusive? so as we compare the difference of the values immediately.

Reply:
Agreed, but I didn't want to create another variable to be able to do the subtraction. It felt trivial enough that it didn't justify another variable.